### DISCUSSION TITLE
Is NativePHP Laravel close to beta release?

### DISCUSSION THREAD

**Discussion by ev3rlost** [NativePHP/laravel#243] [Q&A] [answered]
Is there a roadmap, maybe a due date for a production ready beta release? Maybe with MySQL support?

**Comment by mpociot** [comment]
There's currently no official roadmap we can share. I'm still working on the Electron version and Simon is making great progress on the Tauri part. We will have more updates soon!

**Reply by ev3rlost** [reply]
Thank you for the clarification. Looking forward to new updates!

**Comment by simonhamp** [comment, marked as answer]
I've put together a [Release Roadmap](https://github.com/orgs/NativePHP/projects/2). All **estimates** of course at this stage, but this is a little high-level plan that I've had in my head for a while.
